From 3d8feaa84bb7e9c68205c381369c2f5e43e21eea Mon Sep 17 00:00:00 2001
From: ankris812 <62661743+ankris812@users.noreply.github.com>
Date: Sun, 11 Jun 2023 00:39:38 +0200
Subject: [PATCH] fixed keeloq saving

---
 lib/subghz/protocols/keeloq.c | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/lib/subghz/protocols/keeloq.c b/lib/subghz/protocols/keeloq.c
index 7748da1ee05..a74b1456066 100644
--- a/lib/subghz/protocols/keeloq.c
+++ b/lib/subghz/protocols/keeloq.c
@@ -74,7 +74,7 @@ const SubGhzProtocol subghz_protocol_keeloq = {
     .type = SubGhzProtocolTypeDynamic,
     .flag = SubGhzProtocolFlag_433 | SubGhzProtocolFlag_868 | SubGhzProtocolFlag_315 |
             SubGhzProtocolFlag_AM | SubGhzProtocolFlag_Decodable | SubGhzProtocolFlag_Load |
-            SubGhzProtocolFlag_Send,
+            SubGhzProtocolFlag_Save | SubGhzProtocolFlag_Send,
 
     .decoder = &subghz_protocol_keeloq_decoder,
     .encoder = &subghz_protocol_keeloq_encoder,
@@ -272,10 +272,6 @@ SubGhzProtocolStatus
         subghz_protocol_keeloq_check_remote_controller(
             &instance->generic, instance->keystore, &instance->manufacture_name);
 
-        if(strcmp(instance->manufacture_name, "DoorHan") != 0) {
-            FURI_LOG_E(TAG, "Wrong manufacturer name");
-            ret = SubGhzProtocolStatusErrorParserOthers;
-            break;
         }
 
         //optional parameter parameter
